### ISSUE TITLE
requirejs-rails dependency

### ISSUE THREAD

**Issue by smith** [half-pipe/half-pipe#51]
The generator uses `requirejs_include_tag`, which I thought was provided by [requirejs-rails](https://github.com/jwhitley/requirejs-rails). Is this supposed to be a dependency?

I had to add the latest master of requirejs-rails to my Gemfile in order to get it to work, but now it's giving me a 404 for /javascripts/require.js, presumably because this file is in the vendor/assets of the requirejs-rails gem, and I need to configure something to look there.

Is there something I'm doing wrong, or do you have any advice on how to get this to work? Thanks!

**Comment by joefiorini** [comment]
That is a helper method defined in the previous release. I'd recommend upgrading to the alpha version. We're using it in production currently and others have had luck with it too. It's not well-documented in the readme yet, but I believe release notes on the Releases page cover most of the functionality. Please let me know if you have any questions or troubles.

**Comment by smith** [comment]
Apparently the `reuquirejs_include_tag` was still left over in the generator. I failed to mention I was already running the alpha.

It's kind of confusing that the instructions in the README are not for the version of the software that the README is in. What's going to be the procedure to document 0.3.0 if we're not putting it in the README that is in what will eventually be tagged 0.3.0? Would it make more sense to have the README refer to the version of docs for an older tag instead?

I've opened pull #52 which should fix the problem I had. Thanks!
